stdpopsim/cli.py

```diff
@@ -7,6 +7,7 @@
 import logging
 import platform
 import sys
+import textwrap
 import resource
 
```

Comment:
TODO: need some documentation on the CLI

Reply:
Opened issue to track:  #118